microservices-connector/config/manifests/tgi_nv.yaml

```diff
@@ -23,6 +23,9 @@ data:
   HABANA_LOGS: "/tmp/habana_logs"
```

Comment:
is this a gaudi service? what is nv in tgi_nv.yaml?

Reply:
good catch. We'll clean up later. Let's get this PR in first to unblock the error of running update_manifest.sh

Reply:
PR #567 should fix this